go/vt/vtgate/vindexes/lookup.go

```diff
@@ -44,6 +104,20 @@ type LookupNonUnique struct {
 
 // NewLookup creates a LookupNonUnique vindex.
 func NewLookup(name string, m map[string]string) (Vindex, error) {
+	var fromColumns []string
+	for _, from := range strings.Split(m["from"], ",") {
+		fromColumns = append(fromColumns, strings.TrimSpace(from))
+	}
+	// In the future we can just use MultiColLookupNonUnique for everything.
+	// A vindex with only one column in a special case of the multicol lookup.
+	// However, to reduce risk and don't change so many things at the same time,
+	// I would like to keep the multicolumn implementation separete from the original
+	// one
+	if len(fromColumns) > 1 {
+		lookup := &MultiColLookupNonUnique{name: name}
+		lookup.lkp.Init(m)
+		return lookup, nil
+	}
```

Comment:
I think we should replace the `Lookup...` vindexes with their multi-column versions because one is a complete superset of the other. Otherwise, we'll be unnecessarily be maintaining duplicate functionality.

Reply:
My first inclination was to leave it for a bit of time and make sure we don't discover any regression or odd behavior once this gets to production. With the duplicate path, is very unlikely that we break existent functionality. 

Do you think risk is low and it's worth to refactor in this PR?